package-lock.json

```diff
@@ -12646,8 +12624,7 @@
           "version": "6.0.2",
           "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-6.0.2.tgz",
           "integrity": "sha512-s5kLOcnH0XqDO+FvuaLX8DDjZ18CGFk7VygH40QoKPUQhW4e2rvM0rwUq0t8IQDOwYSeLK01U90OjzBTme2QqA==",
-          "dev": true,
-          "optional": true
```

Comment:
I am not sure if these changes are to be committed to the repo. These packages are optional based on the OS we are trying to run this on. Mostly is an npm issue that it removed a whole bunch of `optional: true`.

@enahum Do you want to remove them?

Reply:
@sudheerDev not really going to remove it, it is being added by npm and it does not cause any issues

Reply:
I see this happening in all repos lately. I don't know which one is valid. Even if i don't change any dependencies and i run `npm install` on web `package-lock` removes these changes similar to the PR.